### DISCUSSION TITLE
시리얼라이즈의 read_only_fields

### DISCUSSION THREAD

**Discussion by devnproyj22** [weaverse-techtide/weaverse-backend#111] [General]
결론적으로, read_only_fields는 역직렬화 과정에서 '사용'되긴 하지만, 그 역할은 해당 필드들을 입력 데이터 처리에서 제외하는 것입니다. 이를 통해 특정 필드들의 값이 클라이언트에 의해 변경되는 것을 방지하고, 데이터의 무결성을 유지할 수 있습니다.

1. 직렬화 (Serialization) 시:
read_only_fields로 지정된 필드들은 모델 인스턴스에서 데이터를 읽어 출력에 포함됩니다.
이 필드들은 API 응답에 포함되어 클라이언트에게 전송됩니다

2. 역직렬화 (Deserialization) 시:

read_only_fields로 지정된 필드들은 입력 데이터에서 무시됩니다.
클라이언트가 이 필드들의 값을 제공하더라도, 시리얼라이저는 이를 처리하지 않습니다.
이 필드들은 create() 또는 update() 메서드에 전달되는 validated_data에 포함되지 않습니다

**Comment by AlbertImKr** [comment]
통일 성 있게 이렇게 하면 어떨까요?
read_only_fields: 
write_only_fields:

**Comment by AlbertImKr** [comment]
자유롭게 하되 Swagger API 문서는 제대로 작동되여야 한다.
